documentation/src/main/asciidoc/userguide/appendices/Configurations.adoc

```diff
@@ -430,8 +430,22 @@ Can reference a
 `FormatMapper` instance,
 `FormatMapper` implementation `Class` reference,
 `FormatMapper` implementation class name (fully-qualified class name) or
-one of the following short hand constants `jackson` or `jsonb`.
-By default the first of the possible providers that is available in the runtime is used, according to the listing order.
+one of the following shorthand constants `jackson` or `jsonb`.
+By default, the first of the possible providers that is available in the runtime is used, according to the listing order.
```

Comment:
This is not related directly to the scope. "in the runtime" is better phrased as "at the runtime"?